### PR TITLE
Dem swimming phantom walls

### PR DESCRIPTION
**Pull request by farrufat-cimne**

### CODE REVIEW

**Comment by farrufat-cimne** [comment]
Both appveyor and travis builds succeeded. Missing a review to finish the merge.

File: applications/DEM_application/python_scripts/main_script.py [hunk 1]
```diff
@@ -458,13 +458,15 @@ def RunMainTemporalLoop(self):
...
 
     def RunAnalytics(self, time, is_time_to_print=True):
-
-        self.MakeAnalyticsMeasurements()
-        if is_time_to_print:
-            self.FaceAnalyzerClass.CreateNewFile()
-            for sp in (sp for sp in self.rigid_face_model_part.SubModelParts if sp[IS_GHOST]):
-                self.face_watcher_analysers[sp.Name].UpdateDataFiles(time)
-            self.FaceAnalyzerClass.RemoveOldFile()
+        for sp in (sp for sp in self.rigid_face_model_part.SubModelParts if sp[IS_GHOST]):
```
Comment (farrufat-cimne) [inline applications/DEM_application/python_scripts/main_script.py]: must check if there is at least one ghost wall before creating class object

**Comment by maceligueta** [comment]
I think that Codacy's suggestions are meaningful. Can someone fix them (@farrufat-cimne , @ipouplana )

**Comment by ipouplana** [comment]
I'll take a look at it @maceligueta

**Review by GuillermoCasas — DISMISSED** [review]

**Comment by GuillermoCasas** [comment]
@maceligueta I hear Travis does not work with numpy. I find this is a real shortcoming, since it is a very basic module of python that we cannot go without. I thought the issue had been solved at some point. What is the status of this question?

**Comment by farrufat-cimne** [comment]
Modifying the code causing the issue when running the unitTest and force those methods to use in-built python functions will avoid the Travis build to keep failing (already tested it). But, as python users, being able to use libraries like numpy is a must.

**Review by maceligueta — APPROVED** [review]

**Pull request merged**